crates/nu_plugin_polars/src/dataframe/lazy/sink.rs

```diff
@@ -0,0 +1,205 @@
+use std::path::PathBuf;
+
+use crate::{
+    values::{cant_convert_err, PolarsPluginObject, PolarsPluginType},
+    PolarsPlugin,
+};
+
+use super::super::values::NuLazyFrame;
+
+use nu_path::expand_path_with;
+use nu_plugin::{EngineInterface, EvaluatedCall, PluginCommand};
+use nu_protocol::{
+    Category, Example, LabeledError, PipelineData, ShellError, Signature, Span, Spanned,
+    SyntaxShape, Type, Value,
+};
+use polars::error::PolarsError;
+use polars_io::{
+    csv::write::CsvWriterOptions, ipc::IpcWriterOptions, json::JsonWriterOptions,
+    parquet::write::ParquetWriteOptions,
+};
+
+#[derive(Clone)]
+pub struct Sink;
+
+impl PluginCommand for Sink {
+    type Plugin = PolarsPlugin;
+
+    fn name(&self) -> &str {
+        "polars sink"
```

Comment:
What about naming it to `polars save`?  To me it seems more consistent with nushell `save` command

Reply:
Sink is the terminology used by pokers. Is different than just saving as it is performing a steaming collect.